docs/07-templates/table.mdx

```diff
@@ -35,6 +35,10 @@ A page with a table is a common pattern in Grafana. This template shows how to u
 - The page actions should be located at the top of the page, above the table.
 - If filters are present, the primary action should be located after the filters, on the right side of the page.
 
+## Visual guidelines
+
+<Image src={'/img/templates/table.png'} alt="Anatomy of the list view" />
```

Comment:
All of the images for grid/table/list show the same "anatomy", is that as intended? 🤔

Reply:
Yes, that guideline applies to all table and list pages.

Reply:
Ah cool 👍